scripts/release.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+if [ $# -eq 0 ]
+  then
+    echo "Missing argument for the new version to release"
+    exit 1
+fi
+
```

Comment:
I would add the following to make sure you don't run the script without `npmrc` installed (because the whole script will run regardless, and this might have unexpected consequences):

```bash
if [ ! $(which npmrc) ]
  then
    echo "You need to install npmrc. Call:"
    echo "  npm install -g npmrc"
    exit 1
fi
```